admin/category_product_listing.php

```diff
@@ -843,7 +847,8 @@
                 $cPath = $product['master_categories_id'];
               }
 
-              if ((!isset($_GET['pID']) && !isset($_GET['cID']) || (isset($_GET['pID']) && ($_GET['pID'] == $product['products_id']))) && !isset($pInfo) && !isset($cInfo) && (substr($action, 0, 3) != 'new')) {
+              if ((!isset($_GET['pID']) && !isset($_GET['cID']) || (isset($_GET['pID']) && ($_GET['pID'] === $product['products_id']))) && !isset($pInfo) && !isset($cInfo) && (strpos($action, 'new')
```

Comment:
Combine `!isset($_GET['pID']) && !isset($_GET['cID'])` into `!isset($_GET['pID'], $_GET['cID'])`

Reply:
This I was leaving separately as I couldn't see if the logic was correct...similarly for the similar test prior to
$cInfo = new objectInfo($category);